Kconfig

```diff
@@ -140,4 +140,27 @@ config BUILD_VM_ROM
 	help
 	  Select if you want to build VM ROM
 
+menu "Performance measurement"
+
+config PERF_MEASURE
+	bool "Enable performance measurement"
+	default n
+	help
+	  Select for performance measurment of various FW components.
+
+config PERF_MEASURE_HISTORY
```

Comment:
I'm still not sure what it is from the description. Is it history length or log2 of history length? If it's LOG2 instead of absolute value it should be named with suffix _LOG2 maybe? We have a lot of variables like that already in xtensa config files f.e. `XCHAL_NUM_AREGS_LOG2`